.github/workflows/cd.yaml

```diff
@@ -0,0 +1,69 @@
+name: Deploy to ECS
+
+on:
+  push:
+    branches:
+      - main
+
+env:
```

Comment:
If these are only used once why not inline them? Also the cluster name and service name, etc aren't needed terraform manages them

Reply:
Also a couple question what is, `DIRECT_URL` and `DATABASE_URL` refers to the hosted supabase db right?